src/main/java/seedu/address/model/person/GraduationDate.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the Graduation Date of a Person in the address book, as combination of a Month and Year
+ * Guarantees: immutable; is valid as declared in {@link #isValidGraduationDate(String)}
+ */
+public class GraduationDate {
+
+    public static final String MESSAGE_CONSTRAINTS = "Graduation Date should be of the format month-year (MM-yyyy)"
+            + " and adhere to the following constraints:\n"
+            + "1. month should contain only numbers, is exactly 2 digit long, and is a valid month number\n"
+            + "2. year should contain only numbers, and is exactly 4 digit long\n"
+            + "3. month and year values are separated by '-'";
```

Comment:
Here also

Reply:
I'm wondering is there a better way to say `separated by '-'`, because the user might misinterpret the separator as '-' instead of hyphen only. Is it better to say `are separated by a hyphen`?

Reply:
Decided to give examples for clarity. Don't think changing "are separated by a hyphen" will add clarity or value to the user